homeassistant/runner.py

```diff
@@ -120,6 +123,16 @@ async def setup_and_run_hass(runtime_config: RuntimeConfig) -> int:
 
 def run(runtime_config: RuntimeConfig) -> int:
     """Run Home Assistant."""
+    if not subprocess._USE_POSIX_SPAWN:  # pylint: disable=protected-access
+        # The subprocess module does not know about Alpine Linux/musl
+        # and will use fork() instead of posix_spawn() which significantly
+        # less efficient. This is a workaround to force posix_spawn()
+        # on Alpine Linux which is supported by musl.
+        _exists = os.path.exists
+        subprocess._USE_POSIX_SPAWN = _exists(  # pylint: disable=protected-access
+            ALPINE_RELEASE_FILE
```

Comment:
I was going back and forth on this check because there could be other cases where they use musl but not alpine. I think that's rare enough that it's not worth the additional overhead of importing the packaging module or requiring it.